### PR TITLE
fix demo descriptions

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
Fix descriptions for the demos:

- [x] 3d_segmentation_demo
- [x] action_recognition_demo
- [x] bert_question_answering_demo

### CODE REVIEW

File: demos/3d_segmentation_demo/python/README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Upon the start-up, the demo reads command-line parameters and loads a network an
 
 ## Preparing to Run
 
+The demo dependencies should be installed before run. That can be achieved with the following command:
+
+```sh
+python3 -m pip install -r <omz_dir>/demos/3d_segmentation_demo/python/requirements.txt
```
Comment (eaidova) [inline demos/3d_segmentation_demo/python/README.md:16]: should it be aligned with other pip instruction format in the repo, e.g. downloader uses
```sh
python3 -mpip install --user -r ...
```

Comment (vladimir-dudnik) [reply demos/3d_segmentation_demo/python/README.md]: changed

**Review by eaidova — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
@eizamaliev ping

**Review by eizamaliev — APPROVED** [review]

File: demos/tests/cases.py [hunk 2]
```diff
@@ -448,6 +448,12 @@ def single_option_cases(key, *args):
         TestCase(options={'-i': 'https://en.wikipedia.org/wiki/OpenVINO',
                           '--questions': ['What frameworks does OpenVINO support?', 'Who are developers?']}),
         [
+            TestCase(options={
+                '-m': ModelArg('bert-large-uncased-whole-word-masking-squad-0001'),
+                '--input_names': '0,1,2',
+                '--output_names': '3171,3172',
+                '--vocab': str(OMZ_DIR / 'models/intel/bert-large-uncased-whole-word-masking-squad-0001/vocab.txt'),
+            }),
```
Comment (eizamaliev) [inline demos/tests/cases.py:456]:
```suggestion
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: demos/tests/cases.py (hunk 2 above)
Comment (vladimir-dudnik) [reply demos/tests/cases.py:456]: why we need to remove this case?

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**